plotly/plotlyfig_aux/handlegraphics/updateScatter.m

```diff
@@ -122,6 +122,24 @@ function updateScatter(obj,scatterIndex)
     
     %---------------------------------------------------------------------%
     
+    %-scatter z-%
+    if ~verLessThan('matlab','8.4') 
```

Comment:
There is an isHG2 helper function you could use here (basically checks to see if HG2 is enabled, which it is for versions greater than 14a)